### PR TITLE
Cross-build cats-effect module for 2.13

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .sbtopts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-J-Xss4m
```
Comment (ruippeixotog) [inline .sbtopts]: Is there a way to define these only on the cats-effect compilation SBT task?

Comment (ghost) [reply .sbtopts]: I don't believe so. Those need to be set before sbt initialises anyway I guess.

Comment (jcazevedo) [reply .sbtopts]: @dom-nguyen: Do we need to have these settings? I'm able to compile the project locally without them (I'm using [paulp/sbt-extras](https://github.com/paulp/sbt-extras) which provides more sensible defaults), and Travis also seems happy without them (see https://travis-ci.org/pureconfig/pureconfig/builds/585745699).

Comment (ghost) [reply .sbtopts]: Hmm yeah I got an failure earlier in Travis wrt out of Metaspace, but now it's fine. Removed the sbtopts now.

Didn't know about sbt-extras, thanks for that!

**Review by jcazevedo — APPROVED** [review]
@dom-nguyen: Thanks for this!

File: .sbtopts (hunk 1 above)
Comment (ruippeixotog) [reply .sbtopts]: FYI, Travis also uses sbt-extras 👍

**Pull request merged**